charts/replicated-library/templates/lib/_apps.tpl

```diff
@@ -0,0 +1,29 @@
+{{/*
+Renders the app objects into Deployments, DaemonSets, and StatefulSets as required by the chart.
+*/}}
+{{- define "replicated-library.apps" -}}
+  {{- range $name, $app := .Values.apps }}
```

Comment:
```suggestion
  {{- range $name, $appValues := .Values.apps }}
```
Maybe we can just drop this extra mapping and go straight to $appValues?

Reply:
yeah that works